core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -167,6 +168,9 @@ oppia.directive('audioBar', [
           };
 
           $scope.onPlayButtonClicked = function() {
+            if ($scope.selectedLanguage.value === 'en-auto') {
```

Comment:
instead of hard-coding this can you please use `isAutogeneratedLanguageCodeSelected`. In general, firstly try reading the code (at least the file you are working on) maybe you'll find something useful to go with instead of writing new thing every time!

Reply:
ok sir